test/N3Parser-test.js

```diff
@@ -1433,6 +1433,353 @@ describe('Parser', () => {
   describe('A Parser instance for the N3 format', () => {
     function parser() { return new Parser({ baseIRI: BASE_IRI, format: 'N3' }); }
 
+    it('should parse a single triple',
+      shouldParse(parser, '<a> <b> <c>.', ['a', 'b', 'c']));
+
+    it('should not parse a default graph',
+      shouldNotParse(parser, '{}', 'Expected entity but got eof on line 1.'));
```

Comment:
```suggestion
      shouldNotParse(parser, '{}', 'Expected entity but got EOF on line 1.'));
```

Reply:
This is testing and existing error message in the parser.